ts/torch_handler/base_handler.py

```diff
@@ -24,6 +29,33 @@
 
 logger = logging.getLogger(__name__)
 
+# Possible values for backend in utils.py
+def check_pt2_enabled():
+    try:
+        import torch._dynamo
+        pt2_enabled = True
+        if torch.cuda.is_available():
+            # If Ampere enable tensor cores and ideally get yourself an A10G or A100
+            if torch.cuda.get_device_capability() >= (8, 0):
+                torch.backends.cuda.matmul.allow_tf32 = True
```

Comment:
Adding comment to explain why its needed

Reply:
I thought this would be sufficient `# If Ampere enable tensor cores and ideally get yourself an A10G or A100` - lemme know what's not clear

Reply:
Added more detail